README.md

```diff
@@ -25,8 +25,6 @@ sudo apt-get install nodejs cmake build-essential swig python3.8
 sudo npm install --global npm
```

Comment:
"wait x seconds" block isn't working.
Console output:
> No function registered for block type control_wait

Reply:
Reopening b/c block still isn't working. Same output.
"wait_for_milliseconds" and "wait_for_touch" are working though